lnst/Recipes/ENRT/MeasurementGenerators/LinuxPerfMeasurementGenerator.py

```diff
@@ -27,12 +34,30 @@ def generate_perf_measurements_combinations(self, config):
             except FileExistsError:
                 pass
 
+        profiled_cpu_groups: list(list(int)) = []
+        if self.params.get("linuxperf_cpus_override", None):
+            profiled_cpu_groups = self.params.linuxperf_cpus_override
+        else:
+            try:
+                profiled_cpu_groups = self.linuxperf_cpus
+            except AttributeError:
+                profiled_cpu_groups = []
```

Comment:
You can replace this with `getattr`:
```suggestion
            profiled_cpu_groups = getattr(self, "linuxperf_cpus", [])
```

Reply:
Yeah makes sense.